roles/nginxsites/tasks/main.yml

```diff
@@ -10,10 +10,44 @@
   when: >
     (("all" in domains_to_use and "none" in domains_to_skip) or item.0.name in domains_to_use)
     and item.0.name not in domains_to_skip
+    and (item.0.wp.install is undefined or item.0.wp.install == 'singlesite')
   with_nested:
     - "{{ domains }}"
     - "{{ nginx_defaults.single_wp_site }}"
 
+- name: Copy configurations to new domain based configuration. (for a sub-directory multisite install)
+  copy:
+    remote_src: yes
+    src: "{{ item.1.src }}"
+    dest: "{{ nginx_defaults.file_path }}/{{ item.0.name }}.{{ item.1.type }}.conf"
+    force: no
+  when: >
+    (("all" in domains_to_use and "none" in domains_to_skip) or item.0.name in domains_to_use)
+    and item.0.name not in domains_to_skip
+    and item.0.wp.install is defined
+    and item.0.wp.install == 'multisite'
+    and (item.0.wp.network_type is undefined or item.0.wp.network_type == 'subdirectory')
```

Comment:
so does this mean that this task is executed by default for multisites if wp.network_type is undefined? Is that intentional?

Reply:
Ya it looks intentional based on the config for subdomains.  Good plan man!

Reply:
Yes, intentional. I added some comments on those in the `vars-sample.yml`.

I was trying to do the setup for MU through the same tasks in `nginxsites` using loops (to skip duplicate code), but with Ansible, in a nested loop you can't really see `items` of a parent element in the child elements. So I had to triple each task to have one for each WP install type and control them with `when`.